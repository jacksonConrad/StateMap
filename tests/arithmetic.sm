// Arithmetic


int DFA main()	
{
	start	
	{
	int a = 5*5;
	print(a);
	int b = 10/10;
	print(b);
	int c = 10 + 5 + 2 - 10;
	print(c);
	int d = (20*5)/10 + 5;
	print(d);
	int e = 12 % 5;
	print(e);

	return 1;
	}
}
