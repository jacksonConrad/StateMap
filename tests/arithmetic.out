25
1
7
15
2
